scripts/perf/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:buster-20200514-slim
+
+RUN apt-get update \
+    && apt-get install -y \
+        git \
+        nvme-cli \
+        build-essential
+
+
+WORKDIR /root
+
+# Clone the fio repo
+RUN git clone https://github.com/axboe/fio
+RUN cd fio/ && ./configure && make
+
+# Clone the SPDK repo
+RUN git clone https://github.com/spdk/spdk.git
+RUN git -C /root/spdk submodule update --init
+
+## Install dependencies for SPDK
+RUN /root/spdk/scripts/pkgdep.sh
+
+# Clean Up
+RUN apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
each `RUN` in a `Dockerfile` creates a new layer, so even though we're removing the files like in this command by doing `rm -rf ...`, this only affects the last layer. The contents of this folders will be available on the previous layers. In order to avoid this, we can have a single `RUN` command for the entire `Dockerfile`